packages/shared-components/cypress/e2e/header/header.cy.js

```diff
@@ -13,4 +13,41 @@ describe('Header', () => {
     // And I should see the search component
     HeaderSteps.getSearch().should('be.visible');
   });
+
+  describe('OntoLicenseAlert', () => {
```

Comment:
Move this outside in its own directory. Also do the same for the steps related with the license alert component.
Implement separate page for the license alert component and test it in isolation. Extend the header tests to check if the license alert appears there as well.

Reply:
Moved the redirect test into `license-alert` directory and added `steps` file, along with `index.html` and `main.js`. The show/hide tests are left here. Also added assertions for the tooltip in this file. The reason, they are here is because I had Issues with the tooltip in `license-alert.cy.js`. For some reason the tests broke there due to the tooltip